COMP 120 Tinkering Graphics-JH182233/Program/Option.py

```diff
@@ -0,0 +1,205 @@
+import time
+
+
+
+#This is the stored pictures, calling "global" so the entire program can use them.
+setMediaPath("F:\COMP 120 Tinkering Graphics-JH182233\Pictures")
+global picture1 
+picture1 = makePicture("Tree-Four-Seasons.jpg")
+global picture2 
+picture2 = makePicture("Tree-Winter.jpg")
+global picture3 
+picture3 = makePicture("Tree-Spring.jpg")
+global picture4 
+picture4 = makePicture("Tree-Summer.jpg")
+global picture5 
+picture5 = makePicture("Tree-Autumn.jpg")  
+global picture6
+picture6 = makePicture("tree.jpg")
+global picture7
+picture7 = makePicture("test.jpg")
+global picM
+picM = makeEmptyPicture(513, 556, black)
+
+
+
+#The main function for the program
+def main():
+#create an empty picture and populate it with others 
+    startX = 0
+    startY = 0
+    global pic
```

Comment:
The variable pic doesn't really need to be global. It would make the code more maintainable and readable to pass it as an argument to any functions that need to use it.